docs/runserver_plus.rst

```diff
@@ -100,7 +100,7 @@ runserver_plus also supports SSL, so that you can easily debug bugs that pop up
 when https is used. To use SSL simply provide a file name for certificates;
 a key and certificate file will be automatically generated::
 
-  $ python manage.py runserver_plus --cert cert
+  $ python manage.py runserver_plus --cert-file cert
```

Comment:
Think we should use `--cert-file cert.crt` here.

I would prefer we use the full paths since they are more clear and easier to understand. We can explain under `Certificate Paths` that a path to the base name is also accepted as a shorthand.